appium/webdriver/webelement.py

```diff
@@ -60,7 +60,7 @@ def get_attribute(self, name: str) -> Optional[Union[str, Dict]]:
         if attribute_value is None:
             return None
 
-        if isinstance(attribute_value, dict):
+        if isinstance(attribute_value, dict) or isinstance(attribute_value, list):
```

Comment:
I assume the return type must be updated as well and then (probably) mypy output fixed

Reply:
also isinstance may accept a tuple as the second argument